src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -50,10 +50,11 @@ HttpStatusCode mongoSubscribeContext
   const std::vector<std::string>&      servicePathV
 )
 {
-    std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
-    std::string servicePath  = (servicePathV.size() == 0)? "" : servicePathV[0];
+    std::string  notifyFormatAsString  = uriParam[URI_PARAM_NOTIFY_FORMAT];
+    Format       notifyFormat          = stringToFormat(notifyFormatAsString);
+    std::string  servicePath           = (servicePathV.size() == 0)? "" : servicePathV[0];
 
-    LM_T(LmtMongo, ("Subscribe Context Request: notifications sent in '%s' format", notifyFormat.c_str()));
+    LM_T(LmtMongo, ("Subscribe Context Request: notifications sent in '%s' format", notifyFormatAsString.c_str()));
```

Comment:
Should we include a similar trace in the other operations which process notifyFormat?